pkg/service/clientattestation/models.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright Gen Digital Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package clientattestation
+
+type IssuerInteractionValidationConfig struct {
+	AttestationVC interface{}           `json:"attestation_vc"`
+	Metadata      []*CredentialMetadata `json:"metadata"`
+}
+
+type VerifierInteractionValidationConfig struct {
+	AttestationVC interface{}           `json:"attestation_vc"`
+	Metadata      []*CredentialMetadata `json:"metadata"`
+}
+
+type CredentialMetadata struct {
+	CredentialID string   `json:"credential_id"`
+	Types        []string `json:"types"`
```

Comment:
Should this be credential_types, issuer_id, issuance_date, and expiration_date?

Reply:
correct. Will update on next iteration.